README.md

```diff
@@ -360,15 +372,29 @@ Note that WorldBank data is only available at the country level and it's not inc
 table. If no values are reported by WorldBank for the country since 2015, the row value will be
 null.
 
+### WorldPop
+Demographics data extracted from WorldPop, estimating total number of people per region broken down
+by age and sex groupings.
+
+| Name | Type | Description | Example |
+| ---- | ---- | ----------- | ------- |
+| **key** | `string` | Unique string identifying the region | ES |
+| **`${sex}`_`${age_bin}`** | `double` | Total number of people categorized as `${sex}` (`m` or `f`) in age bin `${age_bin}` | 1334716 |
+
+Refer to the [WorldPop documentation](https://www.worldpop.org/geodata/summary?id=24798) for more
+details. This data is normalized into buckets that are consistent with other tables and added into
```

Comment:
Once the normalized tables are added, that will be documented as part of the table docs.

Reply:
If the normalized tables are not yet added, let's change "is normalized" to "will be normalized" to avoid confusion and people looking for data that isn't yet available.

Reply:
Done